Puppetfile

```diff
@@ -60,8 +61,7 @@ mod 'puppetlabs/dummy_service', :latest
 # mod 'puppetlabs/firewall', :latest
 
 # Used by psick::vpn::openvpn
-# mod 'luxflux/openvpn',
-#  :git => 'https://github.com/luxflux/puppet-openvpn'
+mod 'puppet-openvpn', '7.1.0'
```

Comment:
Shouldn't this be `luxflux/puppet-openvpn`?

Reply:
No, we use this module, it provides more flexability